JavaAssignments/src/shraddha/StudentDetails.java

```diff
@@ -0,0 +1,44 @@
+//-----------------------------------------------------------------------------------
+//Assignment: Display name, roll no and marks of the student provided by the user.
+//-----------------------------------------------------------------------------------
+
+package shraddha;
+
+public class StudentDetails {
+
+	static int roll_no, marks;
```

Comment:
Why static variables ?
Consider same program for multiple students.
All students will end up with same name,roll number and marks as static variables are shared resources and have single copy.

Reply:
pushed requested changes to the branch